src/java.naming/share/classes/com/sun/jndi/ldap/Connection.java

```diff
@@ -650,6 +650,19 @@ void cleanup(Control[] reqCtls, boolean notifyParent) {
                     } catch (IOException ie) {
                         if (debug)
                             System.err.println("Connection: problem closing socket: " + ie);
```

Comment:
How about combining the debug statement here with the one below and just print the socket isClosed state, something like
```
if (debug) {
    System.err.println("Connection: problem cleaning-up the connection: " + ie);
    System.err.println("Socket isClosed: " + sock.isClosed());
}
```
The text was changed here since the exception can also be thrown by `unpauseReader`